raiden/network/matrixtransport.py

```diff
@@ -477,7 +477,7 @@ def send_queue(address, events):
             for event in events:
                 self.send_async('', address, _event_to_message(event, node_address))
 
-        for (address, queue_name), events in queueids_to_queues.items():
+        for (address, _queue_name), events in queueids_to_queues.items():
```

Comment:
why the underscore here? Since it's not used? Why not completely replace by underscore? Readability?

Reply:
Yeah, could have gone with the underscore, but this way it's more explicit what's in the variable.